modules/smart-agent_systemd-services/detectors-systemd-services.tf

```diff
@@ -0,0 +1,24 @@
+resource "signalfx_detector" "systemd_services" {
```

Comment:
given that the detector is very basic I would have used the yaml generator to ease the creation and the review.

See https://github.com/claranet/terraform-signalfx-detectors/blob/master/docs/development.md#bootstrap-a-new-module for more information and https://github.com/claranet/terraform-signalfx-detectors/blob/master/modules/smart-agent_system-common/conf/01-cpu.yaml for an example